tests/integration/test_with_portainer_service.py

```diff
@@ -21,9 +65,230 @@ async def test_portainer_connection(
     event_loop: asyncio.AbstractEventLoop,
     portainer_container: tuple[URL, str],
     aiohttp_client_session: ClientSession,
-):
+) -> str:
```

Comment:
this is weird. a test with a str as return value?

Reply:
good catch, thx, fixing